### PR TITLE
Adding small refinements to the dbt codes 

### PR DESCRIPTION
**Pull request by ranaafh**

### CODE REVIEW

File: tenant_1/models/example/sources.yml [hunk 1]
```diff
@@ -2,6 +2,6 @@ version: 2
 
 sources:
  - name: sampled_data
-   schema: dbt_rhashemi
+   schema: tenant_A
```
Comment (nbugden) [inline tenant_1/models/example/sources.yml]:
```suggestion
   schema: tenant_1
```

File: tenant_2/dbt_project.yml [hunk 2]
```diff
@@ -2,12 +2,12 @@
 # Name your project! Project names should contain only lowercase characters
 # and underscores. A good package name should reflect your organization's
 # name or the intended use of these models
-name: 'tenant_A'
+name: 'tenant_B'
```
Comment (nbugden) [inline tenant_2/dbt_project.yml]:
```suggestion
name: tenant_2
```

File: tenant_2/dbt_project.yml [hunk 3]
```diff
@@ -2,12 +2,12 @@
...
 version: '1.0.0'
 config-version: 2
 
 # This setting configures which "profile" dbt uses for this project.
-profile: 'tenant_A'
+profile: 'tenant_B'
```
Comment (nbugden) [inline tenant_2/dbt_project.yml]:
```suggestion
profile: tenant_2
```

File: tenant_2/dbt_project.yml [hunk 4]
```diff
@@ -32,7 +32,7 @@ clean-targets:         # directories to be removed by `dbt clean`
 # directory as views. These settings can be overridden in the individual model
 # files using the `{{ config(...) }}` macro.
 models:
-  tenant_A:
+  tenant_B:
```
Comment (nbugden) [inline tenant_2/dbt_project.yml]:
```suggestion
  tenant_2:
```

File: tenant_2/models/example/sources.yml [hunk 5]
```diff
@@ -2,6 +2,6 @@ version: 2
 
 sources:
  - name: sampled_data
-   schema: dbt_rhashemi
+   schema: tenant_B
```
Comment (nbugden) [inline tenant_2/models/example/sources.yml]:
```suggestion
   schema: tenant_2
```

File: tenant_2/profiles.yml [hunk 6]
```diff
@@ -1,4 +1,4 @@
-tenant_A:
+tenant_B:
```
Comment (nbugden) [inline tenant_2/profiles.yml]:
```suggestion
tenant_2:
```

**Review by nbugden — APPROVED** [review]
Not required, but I'd recommend renaming the tenant in the config to match the folders

File: tenant_2/dbt_project.yml (hunk 2 above)
Comment (ranaafh) [reply tenant_2/dbt_project.yml]: makes sense, I'll commit the suggestions and merge the PR

**Pull request merged**